app/models/mentor/student.rb

```diff
@@ -1,6 +1,7 @@
 module Mentor
   class Student
     include ActiveModel::Model
+    include FieldPopulators::SkillLevelFields
```

Comment:
As I said: dangerzone _(plus it's actually not even used in the view like this)_